sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -92,6 +92,9 @@ public partial class HttpAuthorization
         public static bool operator !=(Azure.HttpRange left, Azure.HttpRange right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial interface IModel
```

Comment:
This name feels very general, and looks like it [exists in other namespaces in .NET](https://learn.microsoft.com/en-us/dotnet/api/?term=imodel).  Would something like IAzureModel be too verbose?